config/samples/aerospikebackup.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: asdb.aerospike.com/v1beta1
+kind: AerospikeBackup
+metadata:
+  labels:
```

Comment:
Any specific reason to add labels here? We haven't added these in aerospike CR samples.

Reply:
Not really needed. They are scaffolded by kubebuilder. Will remove